Register admin for remaining models

We haven't registered all project models with the Django Admin, making it difficult to manage content in a centralized manner.

## Task

For each model in our project,
- [ ] register the model with the Django Admin, if not already registered

## Instructions

Start by reading the following guides.

- Django [working with the Django admin](https://docs.djangoproject.com/en/4.1/ref/contrib/admin/) - necessary knowledge to create a pull request with your changes
- GitHub [about working with branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches) - necessary knowledge for registering Django models with the Django admin site

The only files that should be changed are admin.py in each app. Look in the models.py for each app and compare it with the admin.py in the same app. Find any models that have not been registered in admin.py and add them.

## Resources

- https://docs.djangoproject.com/en/4.1/ref/contrib/admin/#modeladmin-objects

Hello @brylie @PrakharMundra  I'm having issues with committing my changes I have modified the pyproject.toml file but I'm still getting an error message 


project\common\utils.py:3: error: Library stubs not installed for "requests" (or incompatible with Python 3.10)
project\common\utils.py:3: note: Hint: "python3 -m pip install types-requests"
project\common\utils.py:3: note: (or run "mypy --install-types" to install all missing stub packages)
project\common\utils.py:3: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)


I initially reached about this issue somedays ago but got not response from you.

--- a/project/accounts/admin.py
+++ b/project/accounts/admin.py
@@ -1,29 +1,7 @@
-from categories.models import Category
 from django.contrib import admin
-from notification.models import Notification
-from threads.models import (
-    Activity,
-    Civi,
-    CiviImage,
-    Fact,
-    Rationale,
-    Rebuttal,
-    Response,
-    Thread,
-)
 
 from .models import Profile, User
 
 # Register your models here.
 admin.site.register(Profile)
-admin.site.register(Category)
 admin.site.register(User)
-admin.site.register(Notification)
-admin.site.register(Fact)
-admin.site.register(Thread)
-admin.site.register(Civi)
-admin.site.register(Response)
-admin.site.register(CiviImage)
-admin.site.register(Activity)
-admin.site.register(Rebuttal)
-admin.site.register(Rationale)

--- a/project/accounts/admin.py
+++ b/project/accounts/admin.py
@@ -1,8 +1,29 @@
+from categories.models import Category
 from django.contrib import admin
+from notification.models import Notification
+from threads.models import (
+    Activity,
+    Civi,
+    CiviImage,
+    Fact,
+    Rationale,
+    Rebuttal,
+    Response,
+    Thread,
+)
 
 from .models import Profile, User
 
 # Register your models here.
 admin.site.register(Profile)
-
+admin.site.register(Category)
 admin.site.register(User)
+admin.site.register(Notification)
+admin.site.register(Fact)
+admin.site.register(Thread)
+admin.site.register(Civi)
+admin.site.register(Response)
+admin.site.register(CiviImage)
+admin.site.register(Activity)
+admin.site.register(Rebuttal)
+admin.site.register(Rationale)

--- a/project/categories/admin.py
+++ b/project/categories/admin.py
@@ -1,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Category
+
+admin.site.register(Category)

--- a/project/notification/admin.py
+++ b/project/notification/admin.py
@@ -1,0 +1,5 @@
+from django.contrib import admin
+
+from .models import Notification
+
+admin.site.register(Notification)

--- a/project/threads/admin.py
+++ b/project/threads/admin.py
@@ -1,0 +1,21 @@
+from django.contrib import admin
+
+from .models import (
+    Activity,
+    Civi,
+    CiviImage,
+    Fact,
+    Rationale,
+    Rebuttal,
+    Response,
+    Thread,
+)
+
+admin.site.register(Fact)
+admin.site.register(Thread)
+admin.site.register(Civi)
+admin.site.register(Response)
+admin.site.register(CiviImage)
+admin.site.register(Activity)
+admin.site.register(Rebuttal)
+admin.site.register(Rationale)

